app/pages/sections/order-items/catalogue-solutions/dashboard/template.test.js

```diff
@@ -1,13 +1,13 @@
-import { componentTester } from '../../../../test-utils/componentTester';
+import { componentTester } from '../../../../../test-utils/componentTester';
 import manifest from './manifest.json';
 
 const setup = {
   template: {
-    path: 'pages/sections/catalogue-solutions/catalogue-solutions/template.njk',
+    path: 'pages/sections/order-items/catalogue-solutions/dashboard/template.njk',
   },
 };
 
-describe('catalogue-solutions page', () => {
+describe('catalogue-solutions - dashboard page', () => {
```

Comment:
the other dashboard's are all capitals
and cataloguse soltuons should be too

Reply:
yes it should. Anything inconsistencies like this please flag up